bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersTracker.java

```diff
@@ -99,9 +99,14 @@ String role() {
     }
 
     static Optional<AdditionalRequiredReviewers> additionalRequiredReviewers(HostUser botUser, List<Comment> comments) {
+        return additionalRequiredReviewers(botUser, comments, null);
+    }
+
+    static Optional<AdditionalRequiredReviewers> additionalRequiredReviewers(HostUser botUser, List<Comment> comments, Comment exclude) {
```

Comment:
The exclude comment parameter probably isn't the one you are looking for, as the reply marker isn't in the command comment but the reply.

Reply:
Thanks, fixed in 7b17768